lms/models/grouping.py

```diff
@@ -104,9 +106,23 @@ class CanvasGroup(Grouping):
     __mapper_args__ = {"polymorphic_identity": Grouping.Type.CANVAS_GROUP}
 
 
-class BlackBoardGroup(Grouping):
+class BlackboardGroup(Grouping):
     __mapper_args__ = {"polymorphic_identity": Grouping.Type.BLACKBOARD_GROUP}
 
 
 class Course(Grouping):
     __mapper_args__ = {"polymorphic_identity": Grouping.Type.COURSE}
+
+
+class GroupingMembership(CreatedUpdatedMixin, BASE):
```

Comment:
No `id` primary_key here, user_id & grouping_id together are the primary key, which I think is nicer here that adding `id` as primary key + adding a unique constraint for those together.